src/views/Trustees.tsx

```diff
@@ -10,12 +10,13 @@ import ScrollToTop from '../components/ScrollToTop';
 import TrusteeRepository from '../services/TrusteeRepository';
 import { TrusteeProps } from '@tpr/layout/lib/components/cards/trustee/trusteeMachine';
 const Trustees = () => {
-  const submit = () => {
+  const onSubmit = (values: any) => {
     console.log('Submitting form');
   };
 
-  const [complete, setComplete] = useState(false);
+  const [complete] = useState(false);
```

Comment:
for **useState** we need 2 properties **[**_the_variable_name_**,** _method_that_modifies_the_variable_**]** 
If "complete" won't change its value, we could simply use `const complete = false;`

Reply:
I removed `setComplete` as this was causing a warning which would stop GitHub Pages from being built. I will need the `useState` hook in the future but for now I will change to `const complete= false;`

Reply:
probably could be a good idea to add a comment stating that it will need to be changed to useState in the future. 
That way we'll know exactly what to do.